zeyple/zeyple.py

```diff
@@ -81,27 +105,30 @@ def _send_message(self, message, recipient):
         """Sends the given message through the SMTP relay"""
         logging.info("Sending message %s", message['Message-id'])
 
-        smtp = smtplib.SMTP(self._config.get('relay', 'host'),
-                            self._config.get('relay', 'port'))
+        smtp = smtplib.SMTP(self.config.get('relay', 'host'),
+                            self.config.get('relay', 'port'))
 
         smtp.sendmail(message['From'], recipient, message.as_string())
         smtp.quit()
 
         logging.info("Message %s sent", message['Message-id'])
 
-    def _load_configuration(self, filename='zeyple.conf'):
+    def load_configuration(self, fname):
```

Comment:
Not a big fan of using abbreviations in programming in general. Without context, it's hard to figure out what is `fname` - is it a _function name_, a _file name_, a _first name_, etc.? And most importantly, when you want to use it, you have to ask yourself _"Did I abbreviate this variable or not? Oh right, I did, but was it `fname` or `fn` or maybe something else?"_.